src/main/scala/org/constellation/domain/transaction/TransactionService.scala

```diff
@@ -96,6 +97,42 @@ class TransactionService[F[_]: Concurrent](
           else Sync[F].unit
       )
 
+  def countNotDummy(status: ConsensusStatus): F[Long] = {
+    val predicate: TransactionCacheData => Boolean = !_.transaction.isDummy
+
+    status match {
+      case ConsensusStatus.Pending     => pending.count(predicate)
+      case ConsensusStatus.InConsensus => inConsensus.count(predicate)
+      case ConsensusStatus.Accepted    => accepted.count(predicate)
+      case ConsensusStatus.Unknown     => unknown.count(predicate)
+    }
+  }
+
+  override def getMetricsMap: F[Map[String, Long]] =
+    List(
+      count(ConsensusStatus.Pending),
+      count(ConsensusStatus.InConsensus),
+      count(ConsensusStatus.Accepted),
+      count(ConsensusStatus.Unknown),
+      countNotDummy(ConsensusStatus.Pending),
+      countNotDummy(ConsensusStatus.InConsensus),
+      countNotDummy(ConsensusStatus.Accepted),
+      countNotDummy(ConsensusStatus.Unknown)
+    ).sequence
+      .map(
+        counts => {
+          Map(
+            "pending" -> counts.get(0).getOrElse(0),
+            "inConsensus" -> counts.get(1).getOrElse(0),
+            "accepted" -> counts.get(2).getOrElse(0),
+            "unknown" -> counts.get(3).getOrElse(0),
+            "pending_notDummy" -> counts.get(4).getOrElse(0),
+            "inConsensus_notDummy" -> counts.get(5).getOrElse(0),
+            "accepted_notDummy" -> counts.get(6).getOrElse(0),
+            "unknown_notDummy" -> counts.get(7).getOrElse(0)
+          )
+        }
+      )
```

Comment:
How about:

```scala
List(
  "pending" -> count(ConsensusStatus.Pending),
   "inConsensus" -> count(ConsensusStatus.InConsensus),
   ...
   "pending_notDummy" -> countNotDummy(ConsensusStatus.Pending),
   ...
).traverse { case (metric, countsF) => countsF.map(_ -> label) }.map(_.toMap)
```

This `counts.get(0)`, `counts.get(1)`... makes me feel strange 😂